frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/review-and-create-step.tsx

```diff
@@ -119,4 +123,5 @@ type ReviewAndCreateProps = {
   errorMessage: string;
   inProgress: boolean;
   state: State;
+  supportedFeatures?: string[];
```

Comment:
I think it needs to be passed down anyways ?
```suggestion
  supportedFeatures: string[];
```